lib/src/df.dart

```diff
@@ -158,7 +160,7 @@ class DataFrame {
             timestampFormat: timestampFormat);
         df._matrix.data.add(colValues);
       }
-      ++i;
```

Comment:
`i++` is more idiomatic when you don't need to assign the pre-increment value to a variable so I've changed to it.